src/locale/languages/de.js

```diff
@@ -0,0 +1,345 @@
+      markAllAsRead: 'Alle als gelesen markieren',
+    },
+  },
+  search: {
+    main: {
+      repositoryButton: 'Repositorien',
+      userButton: 'Benutzer',
+      searchingMessage: 'Suche nach {{query}}',
+      searchMessage: 'Suche nach allen {{type}}',
+      repository: 'repositorium',
+      user: 'benutzer',
+      noUsersFound: 'Keine Benutzer gefunden :(',
+      noRepositoriesFound: 'Keine Repositorien gefunden :(',
+    },
+  },
+  user: {
+    profile: {
+      userActions: 'Benutzer Aktionen',
+      unfollow: 'Entfolgen',
+      follow: 'Folgen',
+    },
+    repositoryList: {
+      title: 'Repositorien',
+    },
+    followers: {
+      title: 'Follower',
+      text: 'Follower',
+      followsYou: 'Folgt dir',
+    },
+    following: {
+      title: 'Following',
+      text: 'Following',
+      followingYou: 'Folgst du',
+    },
+  },
+  repository: {
+    main: {
+      shareRepositoryTitle: 'Teile {{repoName}}',
+      shareRepositoryMessage: 'Check out {{repoName}} on GitHub. {{repoUrl}}',
+      repoActions: 'Repositorium Aktionen',
+      forkAction: 'Fork',
+      starAction: 'Star',
+      unstarAction: 'Unstart',
+      shareAction: 'Teilen',
+      unwatchAction: 'Unwatch',
+      watchAction: 'Watch',
+      ownerTitle: 'BESITZER',
+      contributorsTitle: 'BEITRAGENDE',
+      noContributorsMessage: 'Keine Beitragenden gefunden',
+      sourceTitle: 'QUELLCODE',
+      readMe: 'README',
+      viewSource: 'Quellcode ansehen',
+      issuesTitle: 'ISSUES',
+      noIssuesMessage: 'Keine Issues',
+      noOpenIssuesMessage: 'Keine offenen Issues',
+      viewAllButton: 'Alle anzeigen',
+      newIssueButton: 'Nees Issue',
+      pullRequestTitle: 'PULL REQUESTS',
+      noPullRequestsMessage: 'Keine Pull Requests',
+      noOpenPullRequestsMessage: 'Keine offenen Pull Requests',
+      starsTitle: 'Sterne',
+      forksTitle: 'Forks',
+      forkedFromMessage: 'geforked von',
+      starred: 'Starred',
+      watching: 'Watching',
+      watchers: 'Watchers',
+    },
+    codeList: {
+      title: 'Code',
+    },
+    issueList: {
+      title: 'Issues',
+      openButton: 'Offen',
+      closedButton: 'Geschlossen',
+      searchingMessage: 'Suche nach {{query}}',
+      noOpenIssues: 'Keine offenen Issues gefunden!',
+      noClosedIssues: 'Keine geschlossenen Issues gefunden!',
+    },
+    pullList: {
+      title: 'Pull Requests',
+      openButton: 'Offen',
+      closedButton: 'Geschlossen',
+      searchingMessage: 'Suche nach {{query}}',
+      noOpenPulls: 'Keine offenen Pull Requests gefunden!',
+      noClosedPulls: 'Keine geschlossenen Pull Requests gefunden!',
+    },
+    pullDiff: {
+      title: 'Diff',
+      numFilesChanged: '{{numFilesChanged}} Dateien',
+      new: 'NEU',
+      deleted: 'GELÖSCHT',
+      fileRenamed: 'Datei ohne Änderungen umbenannt',
+    },
+    readMe: {
+      noReadMeFound: 'Keine README.md Datei gefunden.',
```

Comment:
@FlorianTaut  Please translate `readMeActions` as well.

Reply:
@chinesedfan Good catch! Completely missed that one. Should be fixed now! 😄